manage_user_update.php

```diff
@@ -148,8 +148,9 @@
 
 # Project specific access rights override global levels, hence, for users who are changed
 # to be administrators, we have to remove project specific rights.
-if( ( $f_access_level >= $t_admin_threshold ) && ( !user_is_administrator( $f_user_id ) ) ) {
-	user_delete_project_specific_access_levels( $f_user_id );
+if( ( $f_access_level != $t_old_access_level ) && ( $f_access_level >= $t_admin_threshold )
+	&& ( !user_is_administrator( $f_user_id ) ) ) {
```

Comment:
I would have the && at the end of the previous line, use spaces to line up the remainder of the if statement, and use a single tab for indentation of `if` body.

Reply:
I changed it, not sure if it's exactly what you recommended.
I looked at some code before I did it the previous way and found && at the beginning of lines.
We don't have a clear rule for it.